addons/repair/tests/test_repair.py

```diff
@@ -570,13 +570,19 @@ def test_repair_from_return(self):
         self.assertEqual(return_picking.state, 'done')
 
         res_dict = return_picking.action_repair_return()
-        repair_form = Form(self.env[(res_dict.get('res_model'))].with_context(res_dict['context']))
+        repair_form = Form(self.env[(res_dict.get('res_model'))].with_context(res_dict['context']), view=self.env.ref('repair.view_repair_order_form'))
```

Comment:
You can get the view from `res_dict`  
https://github.com/odoo/odoo/blob/1df706bd754d0b48d23604cd0edd3ecb80c8b3f2/addons/repair/models/stock_picking.py#L164